core/src/main/java/org/apache/carbondata/core/util/CarbonUtil.java

```diff
@@ -3338,4 +3338,20 @@ public static boolean hasEncoding(List<org.apache.carbondata.format.Encoding> en
   public static String generateUUID() {
     return UUID.randomUUID().toString();
   }
+
+  /**
+   * Below method will be used to get the datamap schema name from datamap table name
+   * it will split name based on _ and get the last name
+   * This is only for pre aggregate and timeseries tables
+   *
+   * @param tableName
```

Comment:
please make sure that the comments all are useful and remove the useless ones

Reply:
All the comments are useful.